cmd/fluentd/lib/fluent/plugin/out_loki.rb

```diff
@@ -259,7 +259,12 @@ def record_to_line(record)
           when :key_value
             formatted_labels = []
             record.each do |k, v|
-              formatted_labels.push(%(#{k}="#{v}"))
+              v = v.to_s.gsub(%r{(["\\])}, '\\\\\1')
```

Comment:
Sorry to ask but what is it doing ?

Reply:
Oh right, that should probably have a comment of sorts. It prefixes double quotes and backslashes with a backslash. So escaping of special chars.